src/main/java/com/enderio/base/data/recipe/MaterialRecipes.java

```diff
@@ -125,6 +126,38 @@ private void addAlloys(Consumer<FinishedRecipe> recipeConsumer) {
         makeMaterialRecipes(recipeConsumer, EIOItems.END_STEEL_INGOT.get(), EIOItems.END_STEEL_NUGGET.get(), EIOBlocks.END_STEEL_BLOCK.get());
     }
 
+    private void addIngots(Consumer<FinishedRecipe> recipeConsumer) {
+        MultipleCookingRecipeBuilder
```

Comment:
don't use the MultipleCookingRecipeBuilder for this, use the SimpleCookingRecipeBuilder#smelting instead

Reply:
Thanks for the review and advice